infra/ursa/docker-compose.yml

```diff
@@ -70,12 +70,16 @@ services:
     user: "$UID:$GID"
 
   ursa:
+    # default image name which matches suggested docker run version for cache
+    image: ursa
     build:
       context: ../../.
       dockerfile: Dockerfile
     restart: on-failure
     ports:
```

Comment:
These ports shouldn’t be exposed. Only 6009

Reply:
Ok, I can see that you have removed the ports to the host ( https://github.com/fleek-network/ursa/commit/c9dcab850001f7fc23468a251024255d1552ccca ).

I'll then remove the `make docker-run` from the docs. If not required in dev, might too just be removed to avoid confusion.